tests/DCPS/XTypes/Publisher.idl

```diff
@@ -64,6 +70,12 @@ struct ModifiedNameMutableStruct {
   @id(2) long additional_field_modified;
 };
 
+@topic
+@mutable
+struct NestedMutableStruct {
+  @id(1) ModifiedNameMutableStruct nested_struct;
+};
+
```

Comment:
Correct, that is a new one for https://github.com/objectcomputing/OpenDDS/issues/3780

Reply:
Okay. I think the issue ticket already describes the problem and we don't need to keep this type in the test since the test doesn't do anything with it.